### DISCUSSION TITLE
Will Jetpack 6.X be ported to Kirkstone?

### DISCUSSION THREAD

**Discussion by petertalbanese** [OE4T/meta-tegra#1772] [General]
Looks like Scarthgap is all up to date. Will the JP 6.X changes hit Kirstone as well or will we need to bump up our Yocto release to get it?

Thanks!

**Comment by dwalkes** [comment]
I don't believe there are any plans to backport Jetpack 6 to kirkstone at this time.  See discussion from https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2024-02-08 for background.

**Reply by petertalbanese** [reply]
Gotcha, thanks!
